designer/report-designer-extension-pentaho/source/org/pentaho/reporting/designer/extensions/pentaho/drilldown/swing/SwingRemoteDrillDownUi.java

```diff
@@ -56,6 +61,15 @@
   /** Top=level components container. */
   private JPanel editor;
 
+  /** Server URL input field. */
+  JTextField serverUrlField;
```

Comment:
Is there a specific reason why these fields are package protected? Shouldn't they be private instead? If you needed to access them or their contents, accessors would do that job more safely.